.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Backup files generated by rustfmt
 **/*.rs.bk
+
+# IntelliJ IDEA
+.idea
```

Comment:
Not related to this project, can you remove it?

(you can use `.git/info/exclude` for files like this)

Reply:
https://github.com/orhun/git-cliff/pull/696/commits/b8e3492a71a307da5592b4692d7c687edaf0c0bc